src/libs/ords-pcss-client/ords.pcss.swagger.yaml

```diff
@@ -96,7 +96,49 @@ paths:
           in: query
           type: string
           description: Implicit parameter  
-          pattern: '^[^/]+$'      
+          pattern: '^[^/]+$'
+  '/civil/file-content/document-support/':
+    get:
+      tags:
+      - PCSS-CIVIL
+      security:
+      - basicAuth: []      
+      description: Get the civil file content document support data
+      responses:
+        '200':
+          description: output of the endpoint
+          schema:
+            type: object
+            properties:
+              data:
+                $ref: '#/definitions/civilFileContentDocumentSupportResponse'
+      parameters:
+        - name: civil_document_id
+          in: query
+          type: string
+          description: Implicit parameter
+          pattern: '^[^/]+$'
+  '/civil/file-content/document-issue/':
+    get:
+      tags:
+      - PCSS-CIVIL
+      security:
+      - basicAuth: []     
+      description: Get the civil file content document issue data
+      responses:
+        '200':
+          description: output of the endpoint
+          schema:
+            type: object
```

Comment:
move ref here

Reply:
Went through and did this to any that had the object: and properties: fields as well...